packages/angular-output-target/__tests__/generate-angular-component.spec.ts

```diff
@@ -0,0 +1,310 @@
+import { createComponentTypeDefinition, createAngularComponentDefinition } from '../src/generate-angular-component';
```

Comment:
Do we need a test for when custom elements bundle is not being used? I believe it should show up as `defineCustomElementFn: undefined` in `ProxyCmp`. Example: https://github.com/ionic-team/ionic-framework/blob/main/angular/src/directives/proxies.ts#L82

Or do these changes make `defineCustomElementFn: undefined` no longer needed?

Reply:
This spec file should cover that behavior (custom elements bundle not being used) with the "www output" test suite. 

In these changes, I was able to do away with us writing `defineCustomElementFn: undefined` to file. The util only cares if the prop is defined: https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/angular-output-target/angular-component-lib/utils.ts#L44-L46, so the previous proxy output was redundant/unneeded.